src/sampleList.ts

```diff
@@ -0,0 +1,11 @@
+export const sampleList = [
```

Comment:
I would suggest renaming this to `sampleTodoList` to make it clearer what this list is about